int32 dest_id
---
bool success
string message